enterprise_script_service.gemspec

```diff
@@ -38,7 +38,8 @@ end
 
   spec.add_dependency("msgpack", "~> 1.0")
   spec.add_development_dependency("bundler")
-  spec.add_development_dependency("pry-byebug", "~> 3.4")
+  # Newer versions don't work with Ruby 2.5 and 2.6
+  spec.add_development_dependency("pry-byebug", "~> 3.9.0")
```

Comment:
We should probably drop support for everything older than Ruby 3.1 given that's the latest supported version of Ruby